multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/compose/ComposeLocal.kt

```diff
@@ -44,4 +44,11 @@ val LocalMarkdownTypography = compositionLocalOf<MarkdownTypography> {
  */
 val LocalMarkdownPadding = staticCompositionLocalOf<MarkdownPadding> {
     error("No local Padding")
+}
+
+/**
+ * Local [ImageTransformer] provider
+ */
+val LocalImageTransformer = staticCompositionLocalOf<ImageTransformer> {
+    error("No local ImageLinkTransformer")
```

Comment:
No local `ImageTransformer`

Reply:
We can provide the `ImageTransformerImpl` here and remove the `imageTransformer` parameter from Markdown() composable but doing that if the user wants to use a different imagetransformer  they have to use provide the `imageTransformer` using composition local provider in their code. There is noting wrong with it but I think providing `imageTransformer`  as a parameter is more intuitive.

Reply:
Oh I believe you missunderstood me. I only meant the `error` message. It states: `ImageLinkTransformer` instead of `ImageTransformer `

Reply:
sorry my bad, I had named it `ImageLinkTransformer` initially.